stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/gallery/overview/MediaAttachmentDialogFragment.kt

```diff
@@ -28,18 +27,16 @@ internal class MediaAttachmentDialogFragment : BottomSheetDialogFragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        consumeUserMediaAttachmentsArg()
-
         binding.run {
             closeButton.setOnClickListener {
                 dismiss()
             }
-            title.text = this@MediaAttachmentDialogFragment.title
             mediaAttachmentGridView.setMediaClickListener {
                 imageClickListener.invoke(it)
             }
-
-            mediaAttachmentGridView.setAttachments(userMediaAttachments)
+            viewModel.attachmentGalleryItemsLiveData.observe(viewLifecycleOwner) {
+                mediaAttachmentGridView.setAttachments(it)
```

Comment:
Method reference here seams to be possible too

Reply:
Done